src/components/BorrowerProfile/LendCta.vue

```diff
@@ -376,9 +376,6 @@ export default {
 
 			if (this.status === 'fundraising' && this.numLenders > 0) {
 				this.lenderCountVisibility = true;
-			}
-
-			if (this.lenderCountVisibility && this.matching !== '') {
```

Comment:
This block wasn't needed b/c we can `statScrollAnimation` at the same time we determine `lenderCountVisibility` . Previously this check `(this.lenderCountVisibility && this.matching !== '')` was actually short-circuited b/c `this.matching` was always `undefined` thus we always activate `statScrollAnimation`.